test/core/Do.Test.Architecture/Architecture/Layer/ProvidingConfiguration.cs

```diff
@@ -41,6 +41,16 @@ public class LayerZ : LayerBase<DoA, DoB, DoC>
         public class DoC : PhaseBase { }
     }
 
+    protected class Layer : LayerBase { }
+
+    [Test]
+    public void Layer_id_is_its_name()
+    {
+        ILayer layer = new Layer();
+
+        layer.Id.ShouldBe("Layer");
+    }
+

```

Comment:
this case is also between a test case and its stub classes, keep test cases and their stubs together, and move these to the top